examples/README.md

```diff
@@ -54,6 +54,16 @@ Management](https://github.com/triton-inference-server/server/blob/main/docs/use
 which allows backends to behave in a stateless manner and leave the
 state management to Triton.
 
+Triton also provides an option to create python based backends. These
+backends should implement the
+[`TrtionPythonModel` interface](https://github.com/triton-inference-server/python_backend#usage),
+which could be re-used as a backend by multiple models.
+While the only obligatory function is `execute`,
+you may find it helpful to enhance your implementation by adding ` initialize`,
+`finalize`, and any other helper functions. For examples, please refer to
+the [vLLM backend](https://github.com/triton-inference-server/vllm_backend),
+which provides a common python script to serve models supported by vLLM.
```

Comment:
Is this the source of truth for docs on "Python based backends" ? 

If so, I think we should describe how they work in more detail wherever the source of truth is.  Below are some rough ideas I think we should highlight if we don't already document it somewhere:

- To implement a python-based backend, Triton expects to find a folder in the backends directory (ex: `/opt/tritonserver/backends`) with the corresponding backend name, containing a `model.py` that follows the `[PythonBackend model format](link_to_pb_docs)`. For example, using a python-based backend called `my_python_based_backend` using the default backend directory, Triton would expect to find the full path `/opt/tritonserver/backends/my_python_based_backend/model.py`, and any model of this backend would define the following `config.pbtxt`:
```
# config.pbtxt
backend: "my_python_based_backend"

# any necessary I/O or other config settings
...
```
- Python based backends simply load the python backend under the hood and re-use the `model.py` definition for any models that specify this backend.
- You can use the python model's `auto_complete_config` (link here) function to define the I/O upfront so any `config.pbtxt` using this python-based backend isn't required to specify these settings.
- ...

For an example of this, see the <vllm backend>.

Reply:
I would put this level of detail on the main backend README as perhaps a new section instead of the examples. In the beginning we can add c/c++/python (with python hyperlinked to a new section)

The new section "Python based backends" would cover all the details.

Reply:
Yes, working exactly in this direction for the next revision